cmd/scm/main.go

```diff
@@ -437,9 +437,9 @@ func createTeam(client *scm.SCM) cli.ActionFunc {
 			return cli.NewExitError("team user names must be provided (comma separated)", 3)
 		}
 		opt := &scm.TeamOptions{
-			Organization: &pb.Organization{Path: c.String("namespace")},
-			TeamName:     c.String("team"),
-			Users:        users,
+			OrgPath:  c.String("namespace"),
```

Comment:
Would be nice to have documentation for more of the commands available through the `scm` command. There are some command line examples at the top of the file that can be expanded upon.

Reply:
Is it better to add them to other examples inside the file or to the documentation (we have some there)? Or both?

Reply:
Maybe it could be part of the documentation of the `scm` command, when run with `scm -h`. Not sure how to add such extra documentation/examples though. Right now it is just showing commands and options. Anyway, I don't want to prioritize that at this time, but if you could add a Trello card to the backlog, so we have the idea written down, and we can fix it later, when and if we get time.

But for this particular case, maybe just add the comment / examples inline in the file for now, and we can decide later where scm command docs should reside.